backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/UserService.cs

```diff
@@ -99,5 +99,39 @@ public async Task<ImagePathDto> UploadImageAsync(IFormFile file)
             var userId = _httpContextAccessor.HttpContext.User.GetUid();
             return userId;
         }
+
+        public async Task CreateZoomCredentials(NewCredentialsRequestDto newCredentialsRequestDto)
+        {
+            var user = await _context.Users.Include(u => u.Credentials)
+                .FirstOrDefaultAsync(u => u.Uid == GetCurrentUserId());
```

Comment:
Why didn't you use GetCurrentUserInternalAsync?

Reply:
It doesn't Include credentials